lms/djangoapps/discussion/rest_api/tests/test_serializers.py

```diff
@@ -873,106 +806,6 @@ def save_and_reserialize(self, data, instance=None):
-            'body': ['Test body'],
-            'user_id': [str(self.user.id)],
-            'endorsed': ['True'],
-            'anonymous': ['False'],
-            'anonymous_to_peers': ['False'],
-        }
-
-    def test_create_parent_id_nonexistent(self):
-        self.register_get_comment_error_response("bad_parent", 404)
-        data = self.minimal_data.copy()
-        data["parent_id"] = "bad_parent"
-        context = get_context(self.course, self.request, make_minimal_cs_thread())
-        serializer = CommentSerializer(data=data, context=context)
-        assert not serializer.is_valid()
-        assert serializer.errors == {
-            'non_field_errors': ['parent_id does not identify a comment in the thread identified by thread_id.']
-        }
-
-    def test_create_parent_id_wrong_thread(self):
-        self.register_get_comment_response({"thread_id": "different_thread", "id": "test_parent"})
-        data = self.minimal_data.copy()
-        data["parent_id"] = "test_parent"
-        context = get_context(self.course, self.request, make_minimal_cs_thread())
-        serializer = CommentSerializer(data=data, context=context)
-        assert not serializer.is_valid()
-        assert serializer.errors == {
-            'non_field_errors': ['parent_id does not identify a comment in the thread identified by thread_id.']
-        }
-
-    @ddt.data(None, -1, 0, 2, 5)
-    def test_create_parent_id_too_deep(self, max_depth):
-        with mock.patch("lms.djangoapps.discussion.django_comment_client.utils.MAX_COMMENT_DEPTH", max_depth):
-            data = self.minimal_data.copy()
-            context = get_context(self.course, self.request, make_minimal_cs_thread())
-            if max_depth is None or max_depth >= 0:
-                if max_depth != 0:
-                    self.register_get_comment_response({
-                        "id": "not_too_deep",
-                        "thread_id": "test_thread",
-                        "depth": max_depth - 1 if max_depth else 100
-                    })
-                    data["parent_id"] = "not_too_deep"
-                else:
-                    data["parent_id"] = None
-                serializer = CommentSerializer(data=data, context=context)
-                assert serializer.is_valid(), serializer.errors
-            if max_depth is not None:
-                if max_depth >= 0:
-                    self.register_get_comment_response({
-                        "id": "too_deep",
-                        "thread_id": "test_thread",
-                        "depth": max_depth
-                    })
-                    data["parent_id"] = "too_deep"
-                else:
-                    data["parent_id"] = None
-                serializer = CommentSerializer(data=data, context=context)
-                assert not serializer.is_valid()
-                assert serializer.errors == {'non_field_errors': ['Comment level is too deep.']}
-
```

Comment:
[Question, *not* request--I only want to understand]: Why were most of these brought over, but things like `test_create_missing_field` remain?

Reply:
These tests don't rely on the code changes i.e. cs_comments_service to forum_v2 and would still continue to work without having to modify them